src/main/java/io/cdap/plugin/servicenow/sink/model/ServiceNowSchemaField.java

```diff
@@ -19,7 +19,7 @@
 import com.google.gson.annotations.SerializedName;
 
 /**
- * Model class for Schema Field from Schema API
+ * Model class for Schema Field from Column Metadata API
  */
 public class ServiceNowSchemaField {
```

Comment:
There is SchemaField and Service now schema field. 

Please Rename both to indicate what they are dedicated to. 

Example 
MetadataApiSchemaField 
SchemaApiSchemaField ( or anything better )